numba/tests/test_typedlist.py

```diff
@@ -729,6 +729,33 @@ def impl():
         got = impl()
         self.assertEqual(expected, got)
 
+    def test_extend_singleton(self):
```

Comment:
Where's the singleton? Do you mean, "container holding single value"?

Reply:
yes, I was under the impression, that the term 'singleton' could be used to describe such a construct, but perhaps I was mistaken. Is there something shorter than "container holding single value"?

Reply:
It seems, overwhelmingly, singleton refers to the singleton pattern, with the exception of tuple, https://docs.python.org/3/library/stdtypes.html#tuple.

Reply:
Thanks for looking into this. So it seems that use of the term 'singleton' in this context is perhaps permissible. In any case to avoid any future ambiguities I think it will be best to rename the test, how about 'single_value_container'?

Reply:
Sounds good, thanks.

Reply:
done